src/helpers/files.js

```diff
@@ -201,14 +204,19 @@ export const FilesWorkspaceHeader = new Header({
 			vm = new View({
 				propsData: {
 					path: folder.path,
+					hasRichWorkspace,
 				},
 				store,
 			}).$mount(el)
+			console.log('mounted => ' , vm)
 		})
 	},
 
 	updated(folder, view) {
+		const hasRichWorkspace = folder.attributes['rich-workspace-file']
```

Comment:
@skjnldsv Similar problem to the above, but when using bread crumb navigation the file list changes but neither render or updated is being called.

Reply:
Fix in https://github.com/nextcloud/server/pull/40345